tests/unit/test_signers.py

```diff
@@ -32,6 +35,9 @@ def setUp(self):
             'service_name', 'region_name', 'signing_name',
             'v4', self.credentials, self.emitter)
 
+
+class TestSigner(BaseSignerTest):
```

Comment:
May want to add a test to make sure that the newly public properties can be accessed from the request signer (e.g. region name and such).